### PR TITLE
Fix test: test_check_mds_cache_memory_limit

### PR DESCRIPTION
**Pull request by AviadP**
TimeoutSampler added

fix issue #4637 
Signed-off-by: aviadp <apolak@redhat.com>

### CODE REVIEW

File: tests/manage/z_cluster/test_ceph_default_values_check.py [hunk 1]
```diff
@@ -132,13 +133,19 @@ def test_check_mds_cache_memory_limit(self):
...
-            f"mds_cache_memory_limit is set with a value of {expected_mds_value_in_GB}GB"
-        )
+        for sampler in TimeoutSampler(
+            timeout=900, sleep=60, func=get_mds_cache_memory_limit
+        ):
+            if sampler:
+                mds_cache_memory_limit = get_mds_cache_memory_limit()
+                expected_mds_value = 4294967296
+                expected_mds_value_in_GB = int(expected_mds_value / 1073741274)
+                assert mds_cache_memory_limit == expected_mds_value, (
```
Comment (Avilir) [inline tests/manage/z_cluster/test_ceph_default_values_check.py:143]: is it not better to set the values as constants ?

Comment (AviadP) [reply tests/manage/z_cluster/test_ceph_default_values_check.py:143]: agreed, will fix that

**Comment by yitzhak12** [comment]
@AviadP What is the issue we fix here?

**Comment by AviadP** [comment]
> @AviadP What is the issue we fix here?

sometimes test fail to get required information, would like to add some failsafe mechanism

**Review by yitzhak12 — APPROVED** [review]

File: ocs_ci/ocs/constants.py [hunk 2]
```diff
@@ -1396,6 +1396,9 @@
 debug_crush = 20
 """
 
+EXPECTED_MDS_VALUE = 4294967296
```
Comment (Avilir) [inline ocs_ci/ocs/constants.py:1399]: please use 4 * GB

File: ocs_ci/ocs/constants.py [hunk 3]
```diff
@@ -1396,6 +1396,9 @@
...
+EXPECTED_MDS_VALUE_GB = int(EXPECTED_MDS_VALUE / 1073741274)
```
Comment (Avilir) [inline ocs_ci/ocs/constants.py:1400]: please change the `073741274` with `GB`

**Review by Avilir — DISMISSED** [review]

**Review by yitzhak12 — APPROVED** [review]

**Review by ramkiperiy — DISMISSED** [review]

**Review by Avilir — APPROVED** [review]
LGTM

**Review by Akarsha-rai — APPROVED** [review]

**Pull request closed without merging**